docs/docs-components/foundation/typography.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Magnet-UI/docs/documentation.css">
+    <script src="https://kit.fontawesome.com/f4d4a5026f.js" crossorigin="anonymous"></script>
+    <link rel="icon" href="/Magnet-UI/assets/icon.png" type="image/x-icon" />
+    <title>Magnet-UI | Alerts</title>
```

Comment:
You might want to switch the positioning for the title text like <title>Alerts | Magnet-UI </title>, as a user who has accessed your CL would know the name and therefore it would be better to show Alerts first, even if the whole text is not visible, the user will know that they are on the alerts page of your CL.

Reply:
Done [[1495c83](https://github.com/KanekarSnehal/Magnet-UI/pull/12/commits/1495c835388c9d7b611456f6b0a477ff852ed7f2)](url)